src/pages/UserInfoPage/index.js

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import CustomIcon from "assets/svg/CustomIcon";
+import "./UserInfo.scss";
+import { Button, Card, CardContent, Typography } from "@material-ui/core";
+import {
+  retrieveUserInfoRequest,
+  closeUserUpdateModal,
+  openUserUpdateModal,
+} from "./reducer";
+import AddUserInfo from "./AddUserInfo";
+import CustomModal from "../CustomModal";
+
+const UserInfoPage = (props) => {
+  const { userInfo, userInfoUpdateModalStatus } = useSelector(
+    (state) => state.user
+  );
+
+  const dispatch = useDispatch();
+
+  const closeModal = () => {
+    dispatch(closeUserUpdateModal());
+  };
+
+  useEffect(() => {
+    dispatch(retrieveUserInfoRequest());
+  }, [userInfoUpdateModalStatus]);
+
+  const checkKey = ([key, value], i) => {
+    return value !== null && key != "id" ? true : false;
+  };
+
+  const onModal = () => {
+    dispatch(openUserUpdateModal());
+  };
+
+  let picture;
+  var pictureUrl = userInfo["profile_picture_url"];
+  if (pictureUrl) {
+    picture = <img className="profil-picture" src={pictureUrl} />;
+  } else {
+    picture = (
+      <CustomIcon
+        name="empty-user-with-plus"
+        width={100}
+        height={100}
+        className="empty-user"
+      />
+    );
+  }
+
+  var userName = userInfo["username"];
```

Comment:
```suggestion
    const {
    username: userName,
    last_name: lastName,
    first_name: firstName,
    cell_phone: cellPhone,
    email,
    age,
  } = userInfo;
```
Yukaridakini kullanabilirsin